pythonforandroid/recipes/uvloop/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pythonforandroid.recipe import PyProjectRecipe
+
+
+class UvloopRecipe(PyProjectRecipe):
+    version = 'v0.19.0'
+    url = 'git+https://github.com/MagicStack/uvloop'
```

Comment:
I can't remember if it will automatically checkout the tag this way, but if you don't want to pull the full repo history you could try that
```suggestion
    url = 'https://github.com/MagicStack/uvloop/archive/refs/tags/{version}.tar.gz'
```

Reply:
Nice thanks for the insight